docs/utilities/index.rst

```diff
@@ -0,0 +1,10 @@
+Utilities
+========
```

Comment:
Just a minor thing:
reStructuredText underlinings must match the word like this:
```
Utilities
=========
```

Reply:
done